pkg/vmprovider/providers/vsphere/contentlibrary/content_library_provider.go

```diff
@@ -196,10 +196,11 @@ func (cs *provider) RetrieveOvfEnvelopeFromLibraryItem(ctx context.Context, item
 		_ = downloadedFileContent.Close()
 	}()
 
+	// With Image Service validate OVF file, err here can be internet error.
```

Comment:
Nit: I'm not sure if it's accurate to say that the "Image Service" is validating OVF files here. You could just mention that OVF files will be validated before pushing.